packages/dashboard-v2/src/components/CurrentUsage/GraphBar.js

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+
+const Bar = styled.div.attrs({
+  className: `relative flex justify-end h-4 bg-primary rounded-l rounded-r-lg`,
+})`
+  min-width: 1rem;
+  width: ${({ $percentage }) => $percentage}%;
+`;
+
+const BarTip = styled.span.attrs({
+  className: "relative w-4 h-4 border-2 rounded-full bg-white border-primary",
+})``;
+
+const BarLabel = styled.span.attrs({
+  className: "bg-white rounded border-2 border-palette-200 px-3 whitespace-nowrap absolute shadow",
+})`
+  right: max(0%, ${({ $percentage }) => 100 - $percentage}%);
+  top: -0.5rem;
+  transform: translateX(50%);
+`;
+
+export const GraphBar = ({ value, limit, label }) => {
+  const percentage = (value / limit) * 100;
```

Comment:
`nit` cover div by zero here?

Reply:
@peterjan  @kwypchlo How do we want to interpret `0` in terms of usage limit, though? Would it mean user cannot store any data at all, or that the storage is unlimited?

Reply:
let's say it means unlimited